### DISCUSSION TITLE
How far away is the next release?

### DISCUSSION THREAD

**Discussion by semirix** [databendlabs/openraft#305] [General]
Title says it all :sweat_smile: It seems there's been a heap of development and `0.6.4` looks nothing like what's in main.

**Comment by drmingdrmer** [comment]
I'd like the next release will have a stable internal structure(which is undergoing), and the second next release will bring in some API changes for maximizing performance, e.g., avoiding allocation and using references if possible.

Any suggestions?

**Reply by semirix** [reply]
Not any in particular, I was just curious about a rough timeframe. But no worries if you don't have one! The openraft team is doing an awesome job and I'm keen to see how it progresses :)
